source/client/process_impl.cc

```diff
@@ -687,7 +717,12 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const UriPtr& tracing_
           Envoy::Config::Utility::createTagProducer(bootstrap_, envoy_options_.statsTags()));
     }
 
-    createWorkers(number_of_workers_, scheduled_start);
+    absl::Status workers_status = createWorkers(number_of_workers_, scheduled_start);
+    if (!workers_status.ok()) {
+      ENVOY_LOG(error, absl::StrCat("createWorkers failed. Received bad status: ",
+                                    workers_status.message()));
```

Comment:
Nit: Can we use placeholder syntax instead? i.e. ENVOY_LOG(error, "createWorkers failed. Received bad status: {}", worker_status.message);

Reply:
Done. I didn't know the ENVOY_LOG macro supported this.